src/Bit.BlazorUIPlayground.Web/Shared/Header.razor

```diff
@@ -2,6 +2,7 @@
 
 @code {
```

Comment:
plz move this `@code` section to the end of the file and change the content based on what is implemented in the BlazorServer template project.